templates/extra/funding.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h2>How Much Should I Raise?</h2>
+<div style="display:flex; flex-direction: row; justify-content: center; align-items: center">
+    <form>
+        <h4>I want to raise ($)</h4>
+        <input type="text" id="raiseInput" placeholder="$1000" />
+        <h4>Percentage to Investors</h4>
+        <select type="text" id="percentageInput">
+            <option value=".10">10%</option>
+            <option value=".20">20%</option>
+            <option value=".30">30%</option>
+            <option value=".40">40%</option>
+            <option value=".50">50%</option>
+            <option value=".60">60%</option>
+            <option value=".70">70%</option>
+            <option value=".80">80%</option>
+            <option value=".90">90%</option>
+            <option value="1">100%</option>
+        </select>
+        <h4>Amount needed to break even<h4>
+        <h3 class="green-color">
+        $<span id = "result"></span>
```

Comment:
nit: remove extra spaces
```suggestion
        $<span id="result"></span>
```